setup.py

```diff
@@ -1,7 +1,64 @@
-"""Setup script for quconot project

```

Comment:
setup.cfg, setup.py and pyproject.toml seem to share a lot of data. If possible this should be fixed

Reply:
These 3 files essentially have the same function and thus contain the same data. 
setup.py is being phased out. Currently, I have all three files just for compatibility reasons.
The format of the data is a bit different for all. Thus, it can not be changed into a common variable.

Reply:
for now, we should keep all three files. we can transition to modern standard later

Reply:
OK. Make issue for that